### PR TITLE
Verify bitstream writer using svt_reader

### PR DESCRIPTION
**Pull request by Cidana-Developers**
1. Remove aom_reader since svt_reader is ready. The build of
unit test is broken since c7b108ad518b2f223fa7d56c3fdbd482086bc261,
and this CL fixes it.

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -59,6 +60,8 @@ file(GLOB all_files
     "ref/*.h"
     "ref/*.cc"
     "../Source/Lib/Encoder/Codec/*.c"
+    "../Source/Lib/Decoder/Codec/EbDecBitReader.c"
```
Comment (hassount) [inline test/CMakeLists.txt]: seems like a duplication as all c files are added already

Comment (Cidana-Developers) [reply test/CMakeLists.txt]: Actually EbDecBitReader.c is in Decoder.

Comment (hassount) [reply test/CMakeLists.txt]: you're right, my bad

**Comment by kylophone** [comment]
lgtm

**Pull request merged**